src/main/resources/tags/help/ip.tag

```diff
@@ -0,0 +1,9 @@
+type: text
+aliases: address
+
+---
+Geyser is software that you can download and run yourself, so it doesn't have an IP. That being said, there are a few Geyser-related servies that do have an IP:
+
+- There is an _unofficial_ instance of GeyserConnect at `geyserconnect.net` - run `!!geyserconnect` for more info.  This allows you to connect to any server without having to set up Geyser yourself but you may need a Java Edition account for online mode servers (`!!account` for more info). Note that it is HIGHLY DISCOURAGED to join servers such as Hypixel through this as you will likely get banned (see `!!anticheat`).
```

Comment:
Personally, I would just add the tag and nothing about the IP so we don't have the duplicate warnings about Hypixel, etc.

Reply:
You mean make them run !!geyserconnect themselves to find the IP and get all the info? Should I tell them to do it in #bot-spam so the channels don't get clogged?

Reply:
That would be nice.

Reply:
Like the bungee tag (but lets phase out !tag)

> The Geyser and Floodgate BungeeCord plugins only need to be installed on the BungeeCord proxy unless you intend to use the Floodgate API. That way, you need Floodgate on the "backend" servers too.
Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
Run !tag api in #bot-spam for more information regarding the Floodgate API.